syntax = "proto3";
package tts;
option go_package = "pkg/grpc/tts";


service TextToSpeech {
    rpc Synthesize(SynthesizeRequest) returns (SynthesizeResponse) {}
    rpc SynthesizeStream(SynthesizeRequest) returns (stream SynthesizeResponse) {}
}

message SynthesizeRequest {
    string text = 1;
    string language_code = 2;
}


message SynthesizeResponse {
    bytes audio = 1;
}